src/containers/Tenant/Diagnostics/TopShards/TopShards.js

```diff
@@ -32,6 +42,28 @@ function prepareCPUWorkloadValue(value) {
     return `${(value * 100).toFixed(2)}%`;
 }
 
+function prepareDateSizeValue(value) {
+    return new Intl.NumberFormat(Lang.En).format(value);
+}
+
+function stringToDataTableSortOrder(value) {
+    return value && value.split(',').map((columnId) => ({
+        columnId,
+        order: DataTable.DESCENDING,
+    }));
+}
+
+function stringToQuerySortOrder(value) {
+    return value && value.split(',').map((columnId) => ({
+        columnId,
+        order: 'DESC',
+    }));
+}
+
+function dataTableToStringSortOrder(value = []) {
+    return value.map(({columnId}) => columnId).join(',');
+}
+
```

Comment:
why do we hardcode only descending sort order?

Reply:
Ascending order doesn't make sense for the top shards table. The use case is to see the top.
Moreover, if you reverse the sorting of a table which shows 20 items out of 20+ items, the resulting table would contain a different set of rows, which is confusing. One of the solutions is to enumerate rows to show that in ascending order table shows the last 20, and they are different from the first 20. The other one is to disable ascending order, which we decided to do.

Reply:
there is a comment about this below, for the sort handler
I was in doubt where to put it, maybe here would be better?
although keep in mind that code navigation while reviewing differs from the code navigation when writing or reading code in ide